src/Simplex_tree/include/gudhi/Simplex_tree/Simplex_tree_node_explicit_storage.h

```diff
@@ -11,6 +11,13 @@
 #ifndef SIMPLEX_TREE_SIMPLEX_TREE_NODE_EXPLICIT_STORAGE_H_
 #define SIMPLEX_TREE_SIMPLEX_TREE_NODE_EXPLICIT_STORAGE_H_
 
+// Empty base optimization for MSVC - https://learn.microsoft.com/en-us/cpp/cpp/empty-bases
+#if _MSC_VER && !__INTEL_COMPILER
+ #define EMPTY_BASE_CLASS_OPTIMIZATION __declspec(empty_bases)
```

Comment:
Start macro names with `GUDHI_`? We may also want to define it in some common header.

Reply:
I renamed it on e33b19896
Let's see later in which common header it would be the most convenient